02-Reproject-Raster-In-R.Rmd

```diff
@@ -42,6 +43,12 @@ After completing this activity, you will:
 
 * Be able to reproject a raster in R
 
+###Challenge Code
```

Comment:
If most people are finding the lessons via a Google Search it might be best to at least have a brief "find challenge answers in the .R code" even if we do link to a page explaining lessons.   I do think that pretty much everything in the "About" section (what I'm calling the grey box for lack of a better term) can be includes so that we don't have to re-type every time.